packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -791,6 +810,10 @@ export default {
                 data: { ...this.eventData, validationState },
                 logMethod: this.$logger.logWarn
             });
+
+            this.invalidFieldsCount = invalidFieldsLength === 1 ?
+                this.$t('errorMessages.singleFieldError') :
+                this.$t('errorMessages.multipleFieldErrors', { invalidFieldsLength });
```

Comment:
This can be extracted into a computed property. And we should probably use a more descriptive name, since it's more a title than just a count, in my humble opinion.